src/editEngine.js

```diff
@@ -20,14 +20,42 @@ const immutableSet = (key, value) =>
   })
 
 export const batchRemove = ({ editInfo = generateInitialEditInfo(), rows }) => {
-  const { history } = editInfo
-  let finalEdit = editInfo
+  const { added, updated, dirtyMap, updatedMap, removed, history, ...rest } = editInfo
```

Comment:
was this not working? or this is for performance reasons? And i would prefer to have this block of code tested

Reply:
it was working, but for large data there were performance issues( stuck the window for minutes ) for all bulkOps